arviz/plots/bpvplot.py

```diff
@@ -134,14 +135,12 @@ def plot_bpv(
         Passed to :meth:`matplotlib.axes.Axes.plot` or
         :meth:`matplotlib.axes.Axes.axhspan` (when ``kind=u_value``
         and ``reference=analytical``).
-    backend_kwargs : bool, optional
+    backend_kwargs : bool, optional 
         These are kwargs specific to the backend being used, passed to
-        :func:`matplotlib.pyplot.subplots` or
-        :func:`bokeh.plotting.figure`. For additional documentation
-        check the plotting method of the backend.
-    group : {"prior", "posterior"}, optional
-        Specifies which InferenceData group should be plotted. Defaults to 'posterior'.
-        Other value can be 'prior'.
+        :func:`matplotlib.pyplot.subplots` or :func:`bokeh.plotting.figure`.
+        For additional documentation check the plotting method of the backend.
+    group : {"prior", "posterior"}, optional default "posterior"
+        Specifies which InferenceData group should be plotted. 
```

Comment:
This is not really accurate. If `posterior`, then the values in `posterior_predictive` group are compared to the ones in `observed_data`, if `prior` then the same comparison happens, but with the values in `prior_predictive` group.